### DISCUSSION TITLE
Tips and Tutorials for new contributors

### DISCUSSION THREAD

**Discussion by jbusecke** [GliderToolsCommunity/GliderTools#60] [Show and tell]
Hi folks I just wanted to use this space to collect some links to tutorials for folks new to Open Source Software Development. 

I found this a few days ago and think it is a helpful resource how to do a [Pull Request to a repository](https://github.com/firstcontributions/first-contributions)

**Comment by soerenthomsen** [comment]
Thank you @jbusecke !

**Comment by soerenthomsen** [comment]
This is helpful too: https://www.atlassian.com/git/tutorials/what-is-version-control 
Thanks @jbusecke for sharing!
